cmd/kubeadm/app/cmd/upgrade/diff.go

```diff
@@ -184,7 +185,9 @@ func runDiff(fs *pflag.FlagSet, flags *diffFlags, args []string, fetchInitConfig
 			Context:  cmdutil.ValueFromFlagsOrConfig(fs, "context-lines", upgradeCfg.Diff.DiffContextLines, flags.contextLines).(int),
 		}
 
-		difflib.WriteUnifiedDiff(flags.out, diff)
+		if err = difflib.WriteUnifiedDiff(flags.out, diff); err != nil {
+			return fmt.Errorf("write unified diff: %w", err)
```

Comment:
```suggestion
		if err = difflib.WriteUnifiedDiff(flags.out, diff); err != nil {
			return errors.Wrap(err, "error writing unified diff")
```
we don't use %w in kubeadm, yet.

Reply:
ok, fixed it.

Reply:
FWIW, I discussed the usage of `%w` with `@thockin` on Slack. It's not always needed, but Tim's suggestion was to not bother with details and to just use it all the time.

Reply:
it's a matter of consistency - errors.Wrap vs %w

Reply:
there have been a few discussions about this already. we should either fork pkg/errors or write a new mini lib.
mainly because fmt.Errorf(....%w) is not a direct replacement for errors.Wrap. Wrap includes a formatting rule how the error is wrapped, while %w is freeform.

Reply:
So the concern is that `%w` might be embedded inside the string instead of using the `<prefix>: %w` form?

I don't see how having a package would help. Either code reviews catch `<some text> %w <some more text` or they don't.

But consistency is important, so feel free to re-add the LGTM.

Reply:
i would find `fmt.Errorf("foo bar: %w)` vs `fmt.Errorf("foo bar:  %w)`  annoying and worthy of a wrapper func.